shell/platform/darwin/ios/framework/Source/accessibility_bridge_test.mm

```diff
@@ -1664,4 +1664,51 @@ - (void)testFlutterSemanticsScrollViewManagedObjectLifecycleCorrectly {
   // flutterSemanticsScrollView) will cause an EXC_BAD_ACCESS.
   XCTAssertFalse([flutterSemanticsScrollView isAccessibilityElement]);
 }
+
+- (void)testPlatformViewDestructorDoesNotCallSemanticsAPIs {
+  class TestDelegate : public flutter::MockDelegate {
+   public:
+    void OnPlatformViewSetSemanticsEnabled(bool enabled) override {
+      set_semantics_enabled_called = true;
```

Comment:
nit: this test would be a bit more robust if instead of having a bool here you had an int that increments.  That way you could assert that the count is "1" instead of "2" after its deconstructed.  This test would also pass if we somehow messed up setting the delegate.